[2022-11-16 14:40:30,458] [builder.py:193]            [INFO]     |====================================================================================================|
[2022-11-16 14:40:30,459] [builder.py:193]            [INFO]     |                            Trakt Watched Collection in Halloween Movies                            |
[2022-11-16 14:40:30,459] [builder.py:193]            [INFO]     |====================================================================================================|
[2022-11-16 14:40:30,459] [builder.py:194]            [INFO]     |                                                                                                    |
[2022-11-16 14:40:30,459] [builder.py:199]            [INFO]     |================================ Building Definition From Templates ================================|
[2022-11-16 14:40:30,459] [builder.py:209]            [DEBUG]    |                                                                                                    |
[2022-11-16 14:40:30,459] [builder.py:210]            [DEBUG]    | Validating Method: variables                                                                       |
[2022-11-16 14:40:30,459] [builder.py:215]            [DEBUG]    |                                                                                                    |
[2022-11-16 14:40:30,459] [meta.py:210]               [DEBUG]    |========================================== Template trakt ==========================================|
[2022-11-16 14:40:30,460] [meta.py:451]               [DEBUG]    |                                                                                                    |
[2022-11-16 14:40:30,464] [meta.py:210]               [DEBUG]    |========================================= Template shared ==========================================|
[2022-11-16 14:40:30,465] [meta.py:451]               [DEBUG]    |                                                                                                    |
[2022-11-16 14:40:30,483] [meta.py:210]               [DEBUG]    |=========================================== Template arr ===========================================|
[2022-11-16 14:40:30,484] [meta.py:451]               [DEBUG]    |                                                                                                    |
[2022-11-16 14:40:30,492] [meta.py:210]               [DEBUG]    |========================================= Template custom ==========================================|
[2022-11-16 14:40:30,493] [meta.py:451]               [DEBUG]    |                                                                                                    |
[2022-11-16 14:40:30,498] [meta.py:210]               [DEBUG]    |======================================= Template translation =======================================|
[2022-11-16 14:40:30,499] [meta.py:451]               [DEBUG]    |                                                                                                    |
[2022-11-16 14:40:30,503] [meta.py:512]               [DEBUG]    |                                                                                                    |
[2022-11-16 14:40:30,503] [meta.py:513]               [DEBUG]    |==================================== Final Template Attributes =====================================|
[2022-11-16 14:40:30,503] [meta.py:514]               [DEBUG]    |                                                                                                    |
[2022-11-16 14:40:30,503] [meta.py:515]               [DEBUG]    | {'trakt_chart': {'chart': 'watched', 'limit': 100}, 'ignore_blank_results': True, 'collection_mode': 'hide', 'sort_title': '!01_Trakt Watched', 'url_poster': 'https://raw.githubusercontent.com/meisnate12/Plex-Meta-Manager-Images/master/chart/Trakt%20Watched.jpg', 'collection_order': 'custom', 'sync_mode': 'sync', 'cache_builders': 1, 'summary': "A collection of Trakt's Weekly Trakt Watched List.", 'name': 'Trakt Watched'} |
[2022-11-16 14:40:30,503] [meta.py:516]               [DEBUG]    |                                                                                                    |
[2022-11-16 14:40:30,504] [builder.py:223]            [INFO]     |=============================== Validating Trakt Watched Attributes ================================|
[2022-11-16 14:40:30,504] [builder.py:226]            [DEBUG]    |                                                                                                    |
[2022-11-16 14:40:30,504] [builder.py:227]            [DEBUG]    | Validating Method: name                                                                            |
[2022-11-16 14:40:30,504] [builder.py:230]            [DEBUG]    | Value: Trakt Watched                                                                               |
[2022-11-16 14:40:30,504] [builder.py:502]            [DEBUG]    |                                                                                                    |
[2022-11-16 14:40:30,504] [builder.py:503]            [DEBUG]    | Validating Method: ignore_blank_results                                                            |
[2022-11-16 14:40:30,504] [builder.py:504]            [DEBUG]    | Value: True                                                                                        |
[2022-11-16 14:40:30,504] [builder.py:516]            [DEBUG]    |                                                                                                    |
[2022-11-16 14:40:30,504] [builder.py:517]            [DEBUG]    | Validating Method: sync_mode                                                                       |
[2022-11-16 14:40:30,504] [builder.py:521]            [DEBUG]    | Value: sync                                                                                        |
[2022-11-16 14:40:30,504] [builder.py:613]            [DEBUG]    |                                                                                                    |
[2022-11-16 14:40:30,505] [builder.py:614]            [DEBUG]    | Validating Method: collection_order                                                                |
[2022-11-16 14:40:30,505] [builder.py:615]            [DEBUG]    | Value: custom                                                                                      |
[2022-11-16 14:40:30,505] [builder.py:646]            [DEBUG]    |                                                                                                    |
[2022-11-16 14:40:30,505] [builder.py:650]            [DEBUG]    | Validating Method: trakt_chart                                                                     |
[2022-11-16 14:40:30,505] [builder.py:651]            [DEBUG]    | Value: {'chart': 'watched', 'limit': 100}                                                          |
[2022-11-16 14:40:30,505] [builder.py:646]            [DEBUG]    |                                                                                                    |
[2022-11-16 14:40:30,505] [builder.py:650]            [DEBUG]    | Validating Method: collection_mode                                                                 |
[2022-11-16 14:40:30,505] [builder.py:651]            [DEBUG]    | Value: hide                                                                                        |
[2022-11-16 14:40:30,505] [builder.py:646]            [DEBUG]    |                                                                                                    |
[2022-11-16 14:40:30,505] [builder.py:650]            [DEBUG]    | Validating Method: sort_title                                                                      |
[2022-11-16 14:40:30,505] [builder.py:651]            [DEBUG]    | Value: !01_Trakt Watched                                                                           |
[2022-11-16 14:40:30,506] [builder.py:646]            [DEBUG]    |                                                                                                    |
[2022-11-16 14:40:30,506] [builder.py:650]            [DEBUG]    | Validating Method: url_poster                                                                      |
[2022-11-16 14:40:30,506] [builder.py:651]            [DEBUG]    | Value: https://raw.githubusercontent.com/meisnate12/Plex-Meta-Manager-Images/master/chart/Trakt%20Watched.jpg |
[2022-11-16 14:40:30,665] [builder.py:646]            [DEBUG]    |                                                                                                    |
[2022-11-16 14:40:30,665] [builder.py:650]            [DEBUG]    | Validating Method: cache_builders                                                                  |
[2022-11-16 14:40:30,665] [builder.py:651]            [DEBUG]    | Value: 1                                                                                           |
[2022-11-16 14:40:30,665] [builder.py:646]            [DEBUG]    |                                                                                                    |
[2022-11-16 14:40:30,665] [builder.py:650]            [DEBUG]    | Validating Method: summary                                                                         |
[2022-11-16 14:40:30,665] [builder.py:651]            [DEBUG]    | Value: A collection of Trakt's Weekly Trakt Watched List.                                          |
[2022-11-16 14:47:21,083] [plex_meta_manager.py:730]  [DEBUG]    | Traceback (most recent call last):                                                                 |
                                                                 |   File "/usr/local/lib/python3.10/site-packages/urllib3/connectionpool.py", line 449, in _make_request
                                                                 |     six.raise_from(e, None)
                                                                 |   File "<string>", line 3, in raise_from
                                                                 |   File "/usr/local/lib/python3.10/site-packages/urllib3/connectionpool.py", line 444, in _make_request
                                                                 |     httplib_response = conn.getresponse()
                                                                 |   File "/usr/local/lib/python3.10/http/client.py", line 1374, in getresponse
                                                                 |     response.begin()
                                                                 |   File "/usr/local/lib/python3.10/http/client.py", line 318, in begin
                                                                 |     version, status, reason = self._read_status()
                                                                 |   File "/usr/local/lib/python3.10/http/client.py", line 279, in _read_status
                                                                 |     line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
                                                                 |   File "/usr/local/lib/python3.10/socket.py", line 705, in readinto
                                                                 |     return self._sock.recv_into(b)
                                                                 | TimeoutError: timed out
                                                                 | 
                                                                 | During handling of the above exception, another exception occurred:
                                                                 | 
                                                                 | Traceback (most recent call last):
                                                                 |   File "/usr/local/lib/python3.10/site-packages/requests/adapters.py", line 489, in send
                                                                 |     resp = conn.urlopen(
                                                                 |   File "/usr/local/lib/python3.10/site-packages/urllib3/connectionpool.py", line 787, in urlopen
                                                                 |     retries = retries.increment(
                                                                 |   File "/usr/local/lib/python3.10/site-packages/urllib3/util/retry.py", line 550, in increment
                                                                 |     raise six.reraise(type(error), error, _stacktrace)
                                                                 |   File "/usr/local/lib/python3.10/site-packages/urllib3/packages/six.py", line 770, in reraise
                                                                 |     raise value
                                                                 |   File "/usr/local/lib/python3.10/site-packages/urllib3/connectionpool.py", line 703, in urlopen
                                                                 |     httplib_response = self._make_request(
                                                                 |   File "/usr/local/lib/python3.10/site-packages/urllib3/connectionpool.py", line 451, in _make_request
                                                                 |     self._raise_timeout(err=e, url=url, timeout_value=read_timeout)
                                                                 |   File "/usr/local/lib/python3.10/site-packages/urllib3/connectionpool.py", line 340, in _raise_timeout
                                                                 |     raise ReadTimeoutError(
                                                                 | urllib3.exceptions.ReadTimeoutError: HTTPConnectionPool(redacted): Read timed out. (read timeout=60)
                                                                 | 
                                                                 | During handling of the above exception, another exception occurred:
                                                                 | 
                                                                 | Traceback (most recent call last):
                                                                 |   File "//plex_meta_manager.py", line 592, in run_collection
                                                                 |     builder = CollectionBuilder(config, metadata, mapping_name, collection_attrs, library=library, extra=output_str)
                                                                 |   File "/modules/builder.py", line 793, in __init__
                                                                 |     self.obj = self.library.get_playlist(self.name) if self.playlist else self.library.get_collection(self.name, force_search=True)
                                                                 |   File "/modules/plex.py", line 831, in get_collection
                                                                 |     cols = self.search(title=str(data), libtype="collection")
                                                                 |   File "/usr/local/lib/python3.10/site-packages/retrying.py", line 49, in wrapped_f
                                                                 |     return Retrying(*dargs, **dkw).call(f, *args, **kw)
                                                                 |   File "/usr/local/lib/python3.10/site-packages/retrying.py", line 212, in call
                                                                 |     raise attempt.get()
                                                                 |   File "/usr/local/lib/python3.10/site-packages/retrying.py", line 247, in get
                                                                 |     six.reraise(self.value[0], self.value[1], self.value[2])
                                                                 |   File "/usr/local/lib/python3.10/site-packages/six.py", line 719, in reraise
                                                                 |     raise value
                                                                 |   File "/usr/local/lib/python3.10/site-packages/retrying.py", line 200, in call
                                                                 |     attempt = Attempt(fn(*args, **kwargs), attempt_number, False)
                                                                 |   File "/modules/plex.py", line 496, in search
                                                                 |     return self.Plex.search(title=title, sort=sort, maxresults=maxresults, libtype=libtype, **kwargs)
                                                                 |   File "/usr/local/lib/python3.10/site-packages/plexapi/library.py", line 1523, in search
                                                                 |     return self._search(key, maxresults, container_start, container_size, **kwargs)
                                                                 |   File "/usr/local/lib/python3.10/site-packages/plexapi/library.py", line 1535, in _search
                                                                 |     subresults = self.fetchItems(key, container_start=container_start,
                                                                 |   File "/usr/local/lib/python3.10/site-packages/plexapi/library.py", line 443, in fetchItems
                                                                 |     data = self._server.query(ekey, params=url_kw)
                                                                 |   File "/usr/local/lib/python3.10/site-packages/plexapi/server.py", line 697, in query
                                                                 |     response = method(url, headers=headers, timeout=timeout, **kwargs)
                                                                 |   File "/usr/local/lib/python3.10/site-packages/requests/sessions.py", line 600, in get
                                                                 |     return self.request("GET", url, **kwargs)
                                                                 |   File "/usr/local/lib/python3.10/site-packages/requests/sessions.py", line 587, in request
                                                                 |     resp = self.send(prep, **send_kwargs)
                                                                 |   File "//plex_meta_manager.py", line 141, in new_send
                                                                 |     return old_send(*send_args, **kwargs)
                                                                 |   File "/usr/local/lib/python3.10/site-packages/requests/sessions.py", line 701, in send
                                                                 |     r = adapter.send(request, **kwargs)
                                                                 |   File "/usr/local/lib/python3.10/site-packages/requests/adapters.py", line 578, in send
                                                                 |     raise ReadTimeout(e, request=request)
                                                                 | requests.exceptions.ReadTimeout: HTTPConnectionPool(redacted): Read timed out. (read timeout=60)
                                                                 | 
[2022-11-16 14:47:21,094] [plex_meta_manager.py:731]  [ERROR]    | Unknown Error: HTTPConnectionPool(redacted): Read timed out. (read timeout=60)
[2022-11-16 14:47:21,094] [plex_meta_manager.py:736]  [INFO]     | 
[2022-11-16 14:47:21,094] [plex_meta_manager.py:737]  [INFO]     |====================================================================================================|
[2022-11-16 14:47:21,094] [plex_meta_manager.py:737]  [INFO]     |                                 Finished Trakt Watched Collection                                  |
[2022-11-16 14:47:21,094] [plex_meta_manager.py:737]  [INFO]     |                                    Collection Run Time: 0:06:50                                    |
[2022-11-16 14:47:21,094] [plex_meta_manager.py:737]  [INFO]     |====================================================================================================|
